templates/federated_reporting/config.sh.mustache

```diff
@@ -20,7 +20,6 @@ __contexts
 __hosts
 __filechangeslog
 __lastseenhosts
-__lastseenhostslogs
```

Comment:
@craigcomstock is this safe to remove without care of the version of the binaries being run?

Reply:
Yes. We haven't release a version yet. So green-field and fine.

Reply:
We need to make sure the instructions and patches for using 3.12.x hubs as feeders reflect this change.

Reply:
https://ci.cfengine.com/job/fr-tests/543/testReport/junit/new_tests_FR_tests_3.12/x_/FR_3_12_x_feeders_test/ to support my words.